rest/admin-ui-ext/src/main/java/org/keycloak/admin/ui/rest/BruteForceUsersResource.java

```diff
@@ -167,6 +169,12 @@ private Stream<BruteUser> toRepresentation(RealmModel realm, UserPermissionEvalu
                     ModelToRepresentation.toBriefRepresentation(user) :
                     ModelToRepresentation.toRepresentation(session, realm, user);
             userRep.setAccess(usersEvaluator.getAccess(user));
+            if (Boolean.parseBoolean(user.getFirstAttribute(TEMP_ADMIN_ATTR_NAME))) {
+                if (userRep.getAttributes() == null) {
+                    userRep.setAttributes(new HashMap<>());
+                }
+                userRep.getAttributes().put(TEMP_ADMIN_ATTR_NAME, Collections.singletonList("true"));
+            }
```

Comment:
@pedroigor we introduced the `temporary_admin` attribute and based on its existence we display a label on couple screens over the admin console. One of them is the users table on the Users screen. The attribute needs to be re-added here, because the UsersDataTable uses the BruteUser type to render users in the table. 

The thing is toBriefRepresentation() is called every time when the table needs to be rendered, but, as you probably know, brief user representation doesn't contain attributes.

What should be the correct approach from your point of view? I guess we can do one of the following:
* handle the single attribute and add it to the BruteUser type manually (like I did in this PR)
* add all the attributes to the brief user representation (@vmuzikar suggestion as it seems to be the most conceptual/generic way)
* use full UserRepresentation to render the UserDataTable (probably the last resort).

Thanks!

Reply:
cc: @jonkoops

Reply:
@Pepo48 I would start with your proposal and return the single attribute. Returning all user attributes to render a list is too much even though we should not expect many of them and potentially impact the API response time/performance.

Another thing is that we should prefer returning user representations from the User Profile SPI to respect the policies/constraints around user attributes. Something we should change in this class as a follow-up, I would say.